### PR TITLE
[fix]: Add redirect from http to https

### PR DESCRIPTION
**Pull request by mecies**
Based on [this article ](https://serversforhackers.com/c/redirect-http-to-https-nginx) I believe that it should work.

I've just noticed that there is a task for that, [link here](https://monterail.atlassian.net/browse/ME-92)

### CODE REVIEW

File: packages/setup/server/templates/nginx.conf.hbs [hunk 1]
```diff
@@ -35,4 +35,6 @@ server {
   pagespeed EnableFilters inline_javascript;
   pagespeed EnableFilters in_place_optimize_for_browser;
   pagespeed EnableFilters extend_cache;
+
+  return 301 https://$host$request_uri;
```
Comment (kaniaorzel) [inline packages/setup/server/templates/nginx.conf.hbs:39]: This not gonna work IMHO, this one generate redirect loop, SSL is added on higher layer. And without condition generate this loop :( You should consider checking PROTO or host,  Similar is here https://stackoverflow.com/questions/1329647/force-ssl-https-with-mod-rewrite/34065445#34065445

Comment (mecies) [reply packages/setup/server/templates/nginx.conf.hbs:39]: I keep on finding snippet that looks like this 
```RewriteEngine On
RewriteCond %{HTTPS} !=on
RewriteRule ^ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]```
and people are saying to add it to .htaccess file, but we do not have this file and I wonder whether should I create one and add it, or is it added in Heroku directly? It's confusing to me 🤔

Comment (kaniaorzel) [reply packages/setup/server/templates/nginx.conf.hbs:39]: Nginx has different config notation. You could use similar to this: https://gist.github.com/beanieboi/ad526faf063181f336a2  and add **server_name** param.

Comment (mecies) [reply packages/setup/server/templates/nginx.conf.hbs:39]: I also thought maybe it is a matter of creating a custom node server for the Next framework that uses server side rendering [heroku-redirect-next-js-react-client-app-http-to-https](https://stackoverflow.com/questions/56851794/heroku-redirect-next-js-react-client-app-http-to-https)

**Pull request closed without merging**